RoslynPluginGenerator/CommandLine/ProcessedArgs.cs

```diff
@@ -4,37 +4,51 @@
 //   Licensed under the MIT License. See License.txt in the project root for license information.
 // </copyright>
 //-----------------------------------------------------------------------
+using NuGet;
 using System;
 
 namespace SonarQube.Plugins.Roslyn.CommandLine
 {
     public class ProcessedArgs
     {
-        private readonly NuGetReference analyzerRef;
+        private readonly string packageId;
+        private readonly SemanticVersion packageVersion;
         private readonly string sqaleFilePath;
         private readonly string language;
         private readonly bool acceptLicenses;
+        private readonly string outputDirectory;
 
-        public ProcessedArgs(NuGetReference analyzerRef, string sqaleFilePath, string language, bool acceptLicenses)
+        public ProcessedArgs(string packageId, SemanticVersion packageVersion, string language, string sqaleFilePath, bool acceptLicenses, string outputDirectory)
```

Comment:
Miscellaneous: I swapped the positions of *language* and *sqaleFilePath* since language is mandatory and the sqale file path isn't. Perhaps I should have moved sqaleFilePath to the end since it is the only optional parameter.